experimental/PropertyDDS/packages/property-shared-tree-interop/src/test/tableExampleSchemaConverter.spec.ts

```diff
@@ -0,0 +1,211 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable prefer-template */
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable import/no-internal-modules */
+
+import { PropertyFactory } from "@fluid-experimental/property-properties";
+import {
+	createSchemaRepository,
+	defaultSchemaPolicy,
+	FieldSchema,
+	ValueSchema,
+} from "@fluid-internal/tree";
+import { brand } from "@fluid-internal/tree/dist/util/brand";
+import { convertPSetSchemaToSharedTreeLls } from "../schemaConverter";
+
+describe("LlsSchemaConverter", () => {
+	let schemaRepository;
+	it("Conversion", () => {
+		register();
+		const tableSdc: any = brand("Test:Table-1.0.0");
+		const [, OptionalFieldKind] = defaultSchemaPolicy.fieldKinds.keys();
+
+		function getRootFieldSchema(): FieldSchema {
+			return {
+				kind: OptionalFieldKind,
+				types: new Set([tableSdc]),
+			};
+		}
+		const rootFieldSchema: FieldSchema = getRootFieldSchema();
+		schemaRepository = createSchemaRepository();
+		convertPSetSchemaToSharedTreeLls(schemaRepository, rootFieldSchema);
+	});
+	it("Enum", () => {
```

Comment:
These tests depend on the previous test. That is a dangerous pattern, since you will get test failures, eg. if you try to debug a single test. Usually it is better to use `before` in such a case.

Reply:
Already fixed.